wit_dashboard/wit-attribution-table.html

```diff
@@ -71,17 +80,41 @@
       properties: {
         attributions: Array,
         models: Array,
+        colors: Object,
       },
       displayNum: function(num) {
         return d3.format('.3f')(num);
       },
       displayAttributionHeader: function(models, index) {
-        let str = 'attribution';
         if (models != null && models.length > 1) {
           const modelName = models[index];
-          str += ` (model ${modelName})`;
+          return `Model ${modelName}`;
+        } else {
+          return 'Attribution';
         }
-        return str;
+      },
+      getStyle: function(num) {
```

Comment:
We have _useLightColor function for example viewer [here](https://github.com/PAIR-code/what-if-tool/blob/7e58b2b531bf8136f0d8ee09ce102c528e5da7e6/wit_dashboard/wit-example-viewer.ts#L466). It may be better to merge this and the other function so that attribution coloring does not diverge in future. What do you think?

Reply:
done!